libethereum/Executive.cpp

```diff
@@ -180,6 +180,11 @@ void Executive::initialize(Transaction const& _transaction)
 {
 	m_t = _transaction;
 
+	BlockHeader bh;
```

Comment:
This constructing of fake header is not so good, you assume that number and gasLimit are the only things that `verifyTransaction` is going to need, i.e. relying on internal implementation of the called function, which can change in the future and break this code...

Ideally we'd like to use the real block header here
@chfast do you have any advice for this? Maybe we could save the block header as `Executive` member?